test_package/embedded/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ add_executable(CreateEmbeddedSource
   CreateEmbeddedSource.cpp
 )
 
-target_link_libraries(CreateEmbeddedSource CONAN_PKG::ZLIB)
+target_link_libraries(CreateEmbeddedSource CONAN_PKG::zlib)
```

Comment:
Update the test_package with the new lowercase convention in conan-center-index.